src/app/CMakeLists.txt

```diff
@@ -362,7 +362,7 @@ if (OS_IS_WIN)
         DEPLOY_TOOL_OPTIONS ${deploy_tool_options_arg}
     )
 
-    install(SCRIPT ${deploy_script})
+   # install(SCRIPT ${deploy_script})
```

Comment:
I think this change is not needed

Reply:
by the way, this thing doesn't work for me, an error, something about two processes conflicting with each other
@cbjeukendrup FYI

Reply:
What error do you have?

Reply:
It works the first time I run it, but then all the others have this error.
Just run in QtCreator

```
CMake Error at C:/Qt/6.9.1/msvc2022_64/lib/cmake/Qt6Core/Qt6CoreDeploySupport.cmake:542 (message):
  Executing C:/Qt/6.9.1/msvc2022_64/bin/windeployqt.exe failed: 1
Call Stack (most recent call first):
  .qt/deploy_qml_app_MuseScoreStudio_29854e050d.cmake:6 (qt6_deploy_runtime_dependencies)
  src/app/cmake_install.cmake:51 (include)
  src/cmake_install.cmake:42 (include)
  cmake_install.cmake:42 (include)


08:38:20: The process "C:\Qt\Tools\CMake_64\bin\cmake.exe" exited with code 1.
Error while building/deploying project MuseScore (kit: Desktop Qt 6.9.1 MSVC2022 64bit)
When executing step "Install"
```